iati/about/templates/about/about_page.html

```diff
@@ -8,119 +8,43 @@
 <div class="hero">
```

Comment:
Comment to note for when refactoring:

The presentation of hero images could be made more DRY.